problems/countThrees.js

```diff
@@ -1,20 +1,26 @@
 /**
-* Counts the number of threes in a given number.
-* @param {number} n - The input number.
-* @returns {number} - Returns how many threes are in the input.
-*
-* ex: countThrees(300)
-* returns: 1
-*
-* ex: countThrees(3203)
-* returns: 2
-*
-* ex: countThrees(47)
-* returns: 0
-*/
-
-function countThrees() {
+ * Counts the number of threes in a given number.
+ * @param {number} n - The input number.
+ * @returns {number} - Returns how many threes are in the input.
+ *
+ * ex: countThrees(300)
+ * returns: 1
+ *
+ * ex: countThrees(3203)
+ * returns: 2
+ *
+ * ex: countThrees(47)
+ * returns: 0
+ */
 
+function countThrees(n) {
+    let counts = {}
+  for (let i = 0; i < arr.length; i++) {
+    let num = arr[i];
+    counts[num] = counts[num] ? counts[num] + 1 : 1;
+  }
```

Comment:
This code creates a counter object for all elements in the array. We only care about how many 3's we see. 
We can key into our object. 
return counts[3]